examples/thermostat/thermostat-common/include/thermostat-delegate-impl.h

```diff
@@ -58,7 +58,7 @@ class ThermostatDelegate : public Delegate
 
     void InitializePendingPresets() override;
 
-    CHIP_ERROR AppendToPendingPresetList(const Structs::PresetStruct::Type & preset) override;
+    CHIP_ERROR AppendToPendingPresetList(const PresetStructWithOwnedMembers & preset) override;
```

Comment:
This looks like bits of https://github.com/project-chip/connectedhomeip/pull/35161.  Needs rebasing to tip?

Reply:
Yeah, #35161 just got merged, so I hit update and the diff is gone.